### PR TITLE
Update debugging-with-gdb-on-linux.rst

### PR DESCRIPTION
**Pull request by rajat2004**
Added example command with default params for launching Copter SITL, also added Mavproxy launch command

### CODE REVIEW

File: dev/source/docs/debugging-with-gdb-on-linux.rst [hunk 1]
```diff
@@ -42,6 +42,18 @@ There are 3 possible ways:
 ::
 
     gdb [gdb options] --args path-to-SITL-firmware(generally in build/sitl/bin) [SITL options]
+    
+Example: The following launches the default Copter SITL (Change the path as required)
+
+::
+
+    gdb --args /home/<USERNAME>/ardupilot/build/sitl/bin/arducopter -S -I0 --home -35.363261,149.165230,584,353 --model + --speedup 1 --defaults /home/<USERNAME>/ardupilot/Tools/autotest/default_params/copter.parm
```
Comment (peterbarker) [inline dev/source/docs/debugging-with-gdb-on-linux.rst]:
```suggestion
    gdb --args $HOME/ardupilot/build/sitl/bin/arducopter -S -I0 --home -35.363261,149.165230,584,353 --model + --speedup 1 --defaults $HOME/ardupilot/Tools/autotest/default_params/copter.parm
```

File: dev/source/docs/debugging-with-gdb-on-linux.rst [hunk 2]
```diff
@@ -42,6 +42,18 @@ There are 3 possible ways:
 ::
 
     gdb [gdb options] --args path-to-SITL-firmware(generally in build/sitl/bin) [SITL options]
+    
+Example: The following launches the default Copter SITL (Change the path as required)
```
Comment (peterbarker) [inline dev/source/docs/debugging-with-gdb-on-linux.rst]:
```suggestion
Example: The following launches the default Copter SITL:
```

File: dev/source/docs/debugging-with-gdb-on-linux.rst [hunk 3]
```diff
@@ -42,6 +42,18 @@ There are 3 possible ways:
...
+
+::
+
+    gdb --args /home/<USERNAME>/ardupilot/build/sitl/bin/arducopter -S -I0 --home -35.363261,149.165230,584,353 --model + --speedup 1 --defaults /home/<USERNAME>/ardupilot/Tools/autotest/default_params/copter.parm
+    
+You can also attach Mavproxy if required with something like
```
Comment (peterbarker) [inline dev/source/docs/debugging-with-gdb-on-linux.rst]:
```suggestion
You can also attach MAVProxy if required:
```

**Comment by rajat2004** [comment]
@peterbarker Thanks for the review! All the points have been addressed

**Comment by rmackay9** [comment]
@peterbarker, you're happy with this?  It's a bit out of my area so I'm not sure..

for me, if I want to use GDB with SITL i use ../Tools/autotest/sim_vehicle.py -G --map --console

**Comment by peterbarker** [comment]
@rmackay9 yes, this just adds examples to the existing invocations.  A future enhancement might be to make it clear that by far-and-away the easiest way to do things is with the `sim_vehicle.py` command-line options.

**Comment by rmackay9** [comment]
ok, merged!

**Pull request merged**